bedrock/base/templates/includes/protocol/footer/footer.html

```diff
@@ -20,7 +20,7 @@ <h5 class="c-footer-heading">
           {% if ftl_has_messages('footer-corporate-blog') %}
             <li><a href="https://blog.mozilla.org/?{{ utm_params }}&utm_content=company" data-link-type="footer" data-link-name="Corporate Blog">{{ ftl('footer-corporate-blog') }}</a></li>
           {% endif %}
-            <li><a href="https://careers.mozilla.org/?{{ utm_params }}&utm_content=company" data-link-type="footer" data-link-name="Careers">{{ ftl('footer-careers') }}</a></li>
```

Comment:
Pendantic check: are these utm_params and utm_content not needed any more?

Reply:
My assumption was no since it's now a bedrock -> bedrock link, but good find. I'll defer to @pmac

Reply:
Sounds like a good assumption then 👍